### PR TITLE
U/wmwv/copy edit 1

### PR DESCRIPTION
**Pull request by wmwv**
Applies some minor copy-editing.  Mostly "it's" -> "its" for the possessive.

### CODE REVIEW

**Review by ConnorStoneAstro — APPROVED** [review]
This looks great! It seems I have a real problem with `its` haha

File: docs/tutorials/CustomModels.ipynb [hunk 1]
```diff
@@ -193,7 +193,7 @@
     "\n",
     "- What was \"sample_image\" in the radial_model function? This is an object for the image that we are currently sampling. You shouldn't need to do anything with it except get the pixelscale.\n",
     "- what else is in \"ap.utils\"? Lots of stuff used in the background by AstroPhot. For now the organization of these is not very good and sometimes changes, so you may wish to just make your own functions for the time being.\n",
-    "- Why the weird way to access the parameters? The self\\[\"variable\"\\].value format was settled on for simplicity and generality. it's not perfect, but it works.\n",
+    "- Why the weird way to access the parameters? The self\\[\"variable\"\\].value format was settled on for simplicity and generality.  It's not perfect, but it works.\n",
```
Comment (ConnorStoneAstro) [inline docs/tutorials/CustomModels.ipynb]: Just noticed these `.ipynb` notebooks have been removed from the AstroPhot repo to go to their own repo. You can just delete the notebooks

File: docs/tutorials/ModelZoo.ipynb [hunk 2]
```diff
@@ -1502,7 +1502,7 @@
    "source": [
     "## Ray Model\n",
     "\n",
-    "A ray model allows the user to break the galaxy up into regions that can be fit separately. There are two basic kinds of ray model: symmetric and asymetric. A symmetric ray model (symmetric_rays = True) assumes 180 degree symmetry of the galaxy and so each ray is reflected through the center. This means that essentially the major axes and the minor axes are being fit separately. For an asymmetric ray model (symmetric_rays = False) each ray is it's own profile to be fit separately. \n",
+    "A ray model allows the user to break the galaxy up into regions that can be fit separately. There are two basic kinds of ray model: symmetric and asymetric. A symmetric ray model (symmetric_rays = True) assumes 180 degree symmetry of the galaxy and so each ray is reflected through the center. This means that essentially the major axes and the minor axes are being fit separately. For an asymmetric ray model (symmetric_rays = False) each ray is its own profile to be fit separately. \n",
```
Comment (ConnorStoneAstro) [inline docs/tutorials/ModelZoo.ipynb]: and here

File: docs/tutorials/GettingStarted.ipynb [hunk 3]
```diff
@@ -198,7 +198,7 @@
     "# Create a target object with specified pixelscale and zeropoint\n",
     "target = ap.image.Target_Image(\n",
     "    data = target_data,\n",
-    "    pixelscale = 0.262, # Every target image needs to know it's pixelscale in arcsec/pixel\n",
+    "    pixelscale = 0.262, # Every target image needs to know its pixelscale in arcsec/pixel\n",
```
Comment (ConnorStoneAstro) [inline docs/tutorials/GettingStarted.ipynb]: and here

File: astrophot/models/parameter_object.py [hunk 4]
```diff
@@ -17,7 +17,7 @@
 
 class Parameter(object):
     """Object for storing model parameters that are to be optimized in the
-    fitting procedure. A value and it's uncertainty are stored as well
+    fitting procedure. A value and its uncertainty are stored as well
```
Comment (ConnorStoneAstro) [inline astrophot/models/parameter_object.py]: the `parameter_object.py` file is removed in the new parameter system. I think you'll need to merge the current main branch to remove the appropriate files

**Review by ConnorStoneAstro — CHANGES_REQUESTED** [review]
Some files have been deleted in the latest update, those need to be synced up before this PR can be merged

**Comment by wmwv** [comment]
Ah, sorry for not getting this rebased correctly before submitting the PR.

**Comment by wmwv** [comment]
> This looks great! It seems I have a real problem with `its` haha

I offer the reassurance that my rate of errors in typing has gotten worse as I've ~gotten older~ gained experienced.

**Review by ConnorStoneAstro — APPROVED** [review]
This looks good!

**Pull request merged**